card/lib/card/error.rb

```diff
@@ -58,6 +58,11 @@ def report
 
     # error whose message can be shown to any user
     class UserError < Error
+      cattr_accessor :user_error_classes
+      self.user_error_classes = [self,
+                                 ActiveRecord::RecordNotFound,
+                                 ActionController::MissingFile,
+                                 ActiveRecord::RecordInvalid]
```

Comment:
This generalization fixed the problem where missing favicon files were returning 500 errors rather than 404 errors in cucumber.